CONTRIBUTING.md

```diff
@@ -0,0 +1,155 @@
+# How to Contribute to Stravalib
+
+**Attention:** This contributing guide provides an high-level overview for contributing 
```

Comment:
a~~n~~ high-level overview ?

Reply:
fixing now! thanks for reviewing and catching these typos!!!